solo/utils/convert_imgfolder_to_h5.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2022 solo-learn development team.
```

Comment:
so we decided to put it here? I would rename `bash_files` to `scripts` and put it there

Reply:
Yeah, ok